.github/workflows/go.yaml

```diff
@@ -0,0 +1,45 @@
+name: Go
+on: [push]
+jobs:
+
+  build:
+    name: Build
+    runs-on: ubuntu-latest
+    steps:
+
+    - name: Setup Go
+      uses: actions/setup-go@v2-beta
+      with:
+        go-version: '^1.14'
+        
+    - name: Install goimports
+      run: go get golang.org/x/tools/cmd/goimports
+
+    - name: Check code
+      uses: actions/checkout@v2
+    
+    - name: goimports
+      run: test -z "$($(go env GOPATH)/bin/goimports -l .)"
+        
+    - name: Fix goimports on fail
+      run: echo Run goimports -w .
+      if: failure()
+
+    - name: License check
+      run: |
+        test "$( \
+          grep -riL "Copyright .* Google LLC" --include=\*.{go,sh} . | \
```

Comment:
This regex seems loose on the copyright year and overly strict on the file extensions.

Reply:
I can make the year part better. What would you recommend to include to files?

Reply:
Looking at https://github.com/GoogleCloudPlatform/golang-samples/blob/master/license_test.go and contrasting to feedback I've received from @tbpg in the past, we should check:

* Dockerfile
* .html
* .go
* .sh

We also need exemptions for several files as noted in the linked test file for auto-generated code.

Instead of `.*` for the regex, can we do `\d\d\d\d` or equivalent?

Reply:
We should include all filetypes then add exceptions for ones we don't need (`go.sum`, `pb.go` (I think)).

`\d\d\d\d` SGTM.

Reply:
I will propose a blacklist, but it is much longer than whitelist which is why I went that route

Reply:
With a blacklist the command would need to be something more like this `grep -riLI "Copyright \d\d\d\d Google LLC" --exclude=\*.{mod,sum,pb.go,md,dot,json,txt,ssml,cfg,csv,svg,LICENSE} .`